json-rpc/docs/type_event.md

```diff
@@ -72,17 +72,23 @@ Event data is serialized into one JSON object with a "type" field to indicate it
 | receiver            | string                   | Hex-encoded address of the account whose balance was credited by this deposit. |
 | metadata            | string                   | An optional field that can contain extra metadata for the event (from RawTransaction metadata??). This information can be used by an off-chain API to implement a sub-addressing scheme for a wallet. |
 
-#### sentpayment
+#### compliancekeyrotation
```

Comment:
where we can find back these 2 new events key? In parentVasp account?

Reply:
These two new event keys can be found in the dual attestation credential, I actually think I need to add these keys to that view. I'll update.